config.go

```diff
@@ -96,6 +123,9 @@ func DefaultConfig() *Config {
 
 // ValidateConfig is used to validate a sane configuration
 func ValidateConfig(config *Config) error {
+	if config.ProtocolVersion > ProtocolVersionMax {
+		return fmt.Errorf("Protocol version cannot be > %d", ProtocolVersionMax)
```

Comment:
Can you include the current protocol version that didn't match from the config in the error message?